jenkins-master/jobs/scripts/workspace/requirements.txt

```diff
@@ -1,3 +1,3 @@
 boto==2.38.0
-mozdownload==1.18
```

Comment:
The remaining import from mozdownload has since been removed. Ignore this comment.